modules/apps/foundation/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/indexing/BaseIndexSearcherTestCase.java

```diff
@@ -143,15 +135,39 @@ private void _assertPagination(int start, int end, int expectedSize)
 		searchContext.setEnd(end);
 		searchContext.setStart(start);
 
-		Hits hits = indexSearcher.search(searchContext, getDefaultQuery());
+		IdempotentRetryAssert.retryAssert(
```

Comment:
Ok, I thought you were asking for checking for instability, which IdempotentRetryAssert is one. If you are not planning to change that, then this pull is fine to me, it does not really change anything for real. Feel free to proceed on this.